resources/assets/js/components/Application/Intro/IntroPage.tsx

```diff
@@ -44,13 +48,25 @@ export const IntroPage: React.FunctionComponent<IntroPageProps> = ({
           currentTitle={intl.formatMessage(stepNames.welcome)}
           steps={makeProgressBarSteps(
             applicationId,
-            application,
+            steps,
             intl,
             "other",
+            false,
           )}
         />
       )}
-      <Intro handleStart={handleContinue} />
+      {showLoadingState && (
+        <h2
+          data-c-heading="h2"
+          data-c-align="center"
+          data-c-padding="top(2) bottom(3)"
+        >
+          {intl.formatMessage(loadingMessages.loading)}
+        </h2>
+      )}
+      {application !== null && job !== null && (
+        <Intro handleStart={handleContinue} />
```

Comment:
The intro page doesn't actually need the application or job, so no need to wait for them to load.